### PR TITLE
chore(vital-flowcontainer,vital-templates): Move flow container variations and search to site level.

### PR DESCRIPTION
**Pull request by wodenx**
## Changes
- Remove flow container variations from default vitalFlowContaienr and apply at site level
- Remove Search from default vitalHeader and apply at site level
- 
## Test Instructions
- Regression testing for flow containers and search on vital-demo

### CODE REVIEW

**Review by dewen — APPROVED** [review]

File: sites/vital-demo/src/templates/_default.jsx [hunk 1]
```diff
@@ -15,12 +15,10 @@
 import { Fragment } from 'react';
 import { graphql } from 'gatsby';
 import { as } from '@bodiless/fclasses';
+import { asFluidToken } from '@bodiless/vital-elements';
```
Comment (VancheeZze) [inline sites/vital-demo/src/templates/_default.jsx:18]: Were there any linting errors? I think `asFluidToken` is not currently used anywhere in this file.

**Review by VancheeZze — DISMISSED** [review]

File: packages/vital-templates/src/components/Page/tokens/vitalPage.ts [hunk 2]
```diff
@@ -39,9 +37,6 @@ const Base = asFluidToken({
   },
   Components: {
     _default: on(GenericTemplateClean)(vitalGenericTemplate.Generic),
-    PDP: on(PDPTemplateClean)(vitalPDPTemplate.Default),
-    Search: on(GenericTemplateClean)(vitalGenericTemplate.Search),
```
Comment (VancheeZze) [inline packages/vital-templates/src/components/Page/tokens/vitalPage.ts:43]: Just making sure it was done intentionally.

File: sites/vital-demo/src/templates/_default.jsx (hunk 1 above)
Comment (wodenx) [reply sites/vital-demo/src/templates/_default.jsx:18]: good catch - our linter doens't lint gatsby templates

File: packages/vital-templates/src/components/Page/tokens/vitalPage.ts (hunk 2 above)
Comment (wodenx) [reply packages/vital-templates/src/components/Page/tokens/vitalPage.ts:43]: yep - we put them back at site level

**Review by marcopagliarulo — DISMISSED** [review]

**Review by VancheeZze — APPROVED** [review]

**Review by dewen — APPROVED** [review]

**Review by marcopagliarulo — APPROVED** [review]

**Pull request merged**